### DISCUSSION TITLE
Question about where to plugin faikin?

### DISCUSSION THREAD

**Discussion by patrickcollins12** [revk/xESP32-Faikout#300] [General]
I have a Daikin model FTXS20KVMA. Where on this indoor unit do I plugin a Faikin? I've attached some images...

![IMG_1565](https://github.com/revk/ESP32-Faikin/assets/942296/2487a342-53f6-4806-a513-27345b23ed7d)
![IMG_1566](https://github.com/revk/ESP32-Faikin/assets/942296/25239867-4cdd-4db9-8b1a-d2a1658bc340)
![IMG_1567](https://github.com/revk/ESP32-Faikin/assets/942296/5251c4cf-b336-4478-9814-e9832212bf25)

**Comment by MUN0X** [comment]
Not familiar with that specific model but it looks a bit like my ftxm.

I think you should be able to open the metalic side panel you see in the 2nd picture.
